src/tcgwars/logic/impl/gen4/LegendsAwakened.groovy

```diff
@@ -292,7 +292,7 @@ public enum LegendsAwakened implements LogicCardInfo {
             attackRequirement {}
             onAttack {
               opp.all.each {
-                flip { damage 50, it }
+                flip 1, {damage 50, it}
```

Comment:
Personal Opinion: I prefer white space surrounding the curly braces. It's easier to read IMO. I'd also suggest against doing these shortened variable names like `atk_pkm` and `atchEnergy` since it makes it harder to find similar usages by using the find functionality of an IDE, browser, or GitHub